client/js/socket-events/init.js

```diff
@@ -23,10 +23,6 @@ socket.on("init", function(data) {
 
 	if (data.networks.length === 0) {
 		sidebar.find(".empty").show();
-
-		$("#footer").find(".connect").trigger("click", {
-			pushState: false,
-		});
```

Comment:
Mind explaining why you could remove this? I'm not seeing it...

Reply:
Same code gets called by the code that opens correct window.